pythia/models/cnn_lstm.py

```diff
@@ -7,16 +7,44 @@
 from pythia.modules.layers import ConvNet, Flatten
 
 
+_TEMPLATES = {
```

Comment:
@asp, I thought about constants like `_QUESTION_VOCAB_SIZE_REG_NAME`, otherwise issue of typos still relevant.